### PR TITLE
[Hao] Updated Wiki Links and Added Script for Future Update

### PR DESCRIPTION
**Pull request by CatherineZM**
- Updated wiki links with 11.5 docs while maintained the links referring to older version
- Except for `Technote: TSA Setup on two Data Centers in an Environment having DB2 HADR with Multiple Standbys` in `clusterManager` with link https://developer.ibm.com/recipes/tutorials/tsa-setup-on-two-data-centers-in-an-environment-having-db2-hadr-with-multiple-standbys/ 
The article has been archived and cannot find related ones

- Modified the script so that it can be used for future updates

### CODE REVIEW

File: docs/hadrConfig.html [hunk 1]
```diff
@@ -1,439 +1,882 @@
...
+      <a name="BLOCKNONLOGGED___Block_Non_Logged_operations">
+       blocknonlogged - Block non logged operations
+      </a>
+     </h2>
+     <p dir="ltr">
+      First available in V95fp4 and V97.
+      <br/>
+      <br/>
+      When database parameter blocknonlogged is set to YES (default NO), tables with "not logged initially" attribute or not logged LOB columns can not created, either by create table or alter table statement. See also
+      <a href="https://www.ibm.com/docs/en/db2/9.7?topic=dcp-blocknonlogged-block-creation-tables-that-allow-non-logged-activity">
```
Comment (rodericus1987) [inline docs/hadrConfig.html:834]: Should use 11.5 version. In general, for definitions, we want to get links target latest version, unless we are specifically talking about an old version. 

There is another link below for autorestart, which needs to target 11.5 as well.

File: docs/hadrTakeover.html [hunk 2]
```diff
@@ -1,325 +1,676 @@
...
+     <ol dir="ltr">
+      <li>
+       Use db2trc to suspend the tcp/ip communication manager from establishing new connections. See the technote here:
+       <a href="http://www-01.ibm.com/support/docview.wss?uid=swg21673436">
+        Preventing application database connections from automatically activating a database after db2start.
+       </a>
+      </li>
+      <li>
+       Configure your clients to wait between connection attempts to reduce the number of incoming connections. The acrRetryInterval variable, set in the db2dsdriver config file, can be used to control client reconnection intervals. See here:
+       <a href="https://www.ibm.com/docs/en/db2/11.1?topic=njcshacd-configuration-db2-automatic-client-reroute-support-applications-other-than-java">
```
Comment (rodericus1987) [inline docs/hadrTakeover.html:569]: Use 11.5 version

File: docs/clusterManagers.html [hunk 3]
```diff
@@ -1,307 +1,737 @@
...
+       </ul>
+      </li>
+     </ul>
+     <h3 dir="ltr">
+      Technote: TSA Setup on two Data Centers in an Environment having DB2 HADR with Multiple Standbys
+     </h3>
+     <p dir="ltr">
+       <!-- This link has been archived. Could not find the related article -->
+      <a href="https://developer.ibm.com/recipes/tutorials/tsa-setup-on-two-data-centers-in-an-environment-having-db2-hadr-with-multiple-standbys/">
+       https://developer.ibm.com/recipes/tutorials/tsa-setup-on-two-data-centers-in-an-environment-having-db2-hadr-with-multiple-standbys/
```
Comment (rodericus1987) [inline docs/clusterManagers.html:716]: This link needs replace.

Comment (CatherineZM) [reply docs/clusterManagers.html:716]: I wasn't able to find a link for  the article. Is there anything else we can replace it with?

Comment (rodericus1987) [reply docs/clusterManagers.html:716]: I think this is close enough, but it's using Pacemaker. 
https://www.ibm.com/docs/en/db2/11.5?topic=hmsd-scenario-deploying-two-sites-multiple-standby-cluster-same-site-failover-automation
For TSA use this one: https://www.idug.org/HigherLogic/System/DownloadDocumentFile.ashx?DocumentFileKey=52df64bc-a067-4fcb-99ff-01781c8836c1

**Comment by CatherineZM** [comment]
@rodericus1987 Addressed comments. Please review again. Thank you!

File: docs/clusterManagers.html [hunk 4]
```diff
@@ -708,12 +708,16 @@ <h1 dir="ltr">
       </li>
      </ul>
      <h3 dir="ltr">
-      Technote: TSA Setup on two Data Centers in an Environment having DB2 HADR with Multiple Standbys
+      TSA Setup on two Data Centers in an Environment having DB2 HADR with Multiple Standbys
```
Comment (rodericus1987) [inline docs/clusterManagers.html]: Automation setup on two Data Centers...
We are now starting on pacemaker and moving away from TSA
There are two links here.

**Review by rodericus1987 — APPROVED** [review]
I made the change myself.

**Pull request merged**